frontend/src/components/Fav.jsx

```diff
@@ -0,0 +1,29 @@
+import logo from "../assets/images/logo_alone.png";
+
+export default function Fav() {
+  const allFav = [
+    { img: logo, name: "Favori 1" },
+    { img: logo, name: "Favori 2" },
+    { img: logo, name: "Favori 3" },
+    { img: logo, name: "Favori 4" },
+    { img: logo, name: "Favori 5" },
+    { img: logo, name: "Favori 6" },
+    { img: logo, name: "Favori 7" },
+    { img: logo, name: "Favori 8" },
+    { img: logo, name: "Favori 9" },
+  ];
+
+  return (
+    <>
+      <h1 className="text-center text-3xl my-16">Favoris</h1>
```

Comment:
put the heading in the section (and so the fragment will be useless)

Reply:
commit [2a59c29](https://github.com/WildCodeSchool/2022-03-JS-Reims-project-3-shop-around-me/pull/34/commits/2a59c294f7b8edc919f3fc026488c2af54ad4c08)